rust-workspace/crates/ntc-oracle-node/src/settings.rs

```diff
@@ -0,0 +1,38 @@
+// https://github.com/mehcode/config-rs/blob/master/examples/hierarchical-env/settings.rs
+
+use std::env;
+
+use config::{Config, ConfigError, File};
+use serde_derive::Deserialize;
+
+#[derive(Debug, Deserialize)]
+pub struct Purestake {
+    pub indexer_url: String,
+    pub indexer_token: String,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct Settings {
+    pub purestake: Purestake,
+}
+
+impl Settings {
+    pub fn new() -> Result<Self, ConfigError> {
+        let run_mode = env::var("RUN_MODE").unwrap_or_else(|_| "development".into());
```

Comment:
It might be safer to deal with this at compile time, using conditional compilation features: https://doc.rust-lang.org/reference/conditional-compilation.html#debug_assertions

Reply:
Code was from example code and I do not use it so I just deleted this :)